hnn_core/gui/_viz_manager.py

```diff
@@ -711,6 +739,7 @@ def compose(self):
         ])
         return config_panel, fig_output_container
 
+    @unlink_relink(attribute='figs_config_tab_link')
```

Comment:
Need to think of one line comment on ipywidgets figure linking logic

Reply:
sorry I missed the discussion on this, is it a bugfix or something related to the new version of ipywidgets?